app/Http/Transformers/PostTransformer.php

```diff
@@ -37,6 +37,7 @@ public function transform(Post $post)
             'type' => $post->type,
             'action' => $post->getActionName(),
             'action_id' => $post->action_id,
+            'campaign_id' => $post->campaign_id,
```

Comment:
is this removed since we have the `action_id` returned?

Reply:
Sorry, could you clarify what you mean by "removed" here?

Reply:
I added the `campaign_id` field to the post transformer because we store it on the post in the database, but do not currently expose it here (and it was handy to be able to easily snag for displaying in this component).

Reply:
oops my goof! I thought this was added instead of removed 🤦‍♀️